libs/langchain/langchain/embeddings/openai.py

```diff
@@ -231,7 +250,7 @@ def build_extra(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         values["model_kwargs"] = extra
         return values
 
-    @root_validator(pre=True)
```

Comment:
do we have to change

Reply:
The other oai types don't do pre, and having pre makes it fail on request_timeout iirc. Can jump through some hoops if we want to keep it

Reply:
changed